.github/workflows/build_tests.yml

```
@@ -0,0 +1,39 @@
+        run: |
+          sudo apt-get update
+          sudo apt install python3-dev swig libssl-dev
+      - name: Build Source Packages
+        run: |
+          python setup.py sdist
+      - name: Build Distribution Packages
+        run: |
+          python setup.py bdist_wheel
+      - name: Install skill
+        run: |
+          pip install .
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)